ark-testing/tests/bark.rs

```diff
@@ -33,3 +35,85 @@ async fn onboard_bark() {
 }
 
 #[tokio::test]
+async fn multiple_round_payments() {
+	// Initialize the test
+	let ctx = TestContext::generate();
+	let bitcoind = ctx.bitcoind("bitcoind-1").await.unwrap();
+	let aspd = ctx.aspd("aspd-1", &bitcoind).await.unwrap();
+
+	// Fund the asp
+	bitcoind.generate(106).await.unwrap();
+	bitcoind.fund_aspd(&aspd, Amount::from_int_btc(10)).await.unwrap();
+
+	// Create a few clients
+	let bark_1 = ctx.bark("bark_1".to_string(), &bitcoind, &aspd).await.unwrap();
+	let bark_2 = ctx.bark("bark_2".to_string(), &bitcoind, &aspd).await.unwrap();
+	let bark_3 = ctx.bark("bark_3".to_string(), &bitcoind, &aspd).await.unwrap();
+	let bark_4 = ctx.bark("bark_4".to_string(), &bitcoind, &aspd).await.unwrap();
+	let bark_5 = ctx.bark("bark_5".to_string(), &bitcoind, &aspd).await.unwrap();
+	let bark_6 = ctx.bark("bark_6".to_string(), &bitcoind, &aspd).await.unwrap();
+	let bark_7 = ctx.bark("bark_7".to_string(), &bitcoind, &aspd).await.unwrap();
+	let bark_8 = ctx.bark("bark_8".to_string(), &bitcoind, &aspd).await.unwrap();
+
+	// Provide onchain funds
+	tokio::join!(
```

Comment:
Volgens mij kunt ge equivalent gewoon `bitcoind.fund_bark(&bark_1, &Amount::from_sat(90_000))` doen zonder de async wrap en await, maar dan me `tokio::try_join!` da een result returnt en early returnt als er een error is ergens. En dan gewoon unwrap na de `try_join`.

Like so
```rust
	tokio::try_join!(
		bitcoind.fund_bark(&bark_1, &Amount::from_sat(90_000)),
		bitcoind.fund_bark(&bark_2, &Amount::from_sat(90_000)),
		bitcoind.fund_bark(&bark_3, &Amount::from_sat(90_000)),
		bitcoind.fund_bark(&bark_4, &Amount::from_sat(90_000)),
		bitcoind.fund_bark(&bark_5, &Amount::from_sat(90_000)),
		bitcoind.fund_bark(&bark_6, &Amount::from_sat(90_000)),
		bitcoind.fund_bark(&bark_7, &Amount::from_sat(90_000)),
		bitcoind.fund_bark(&bark_8, &Amount::from_sat(90_000)),
	).unwrap();
```